frontend/src/member/MemberBoard.tsx

```diff
@@ -25,6 +34,13 @@ export default function MemberCard(props: MemberCardProps) {
                 <StyledMail>
                     {props.member.email}
                 </StyledMail>
+                <StyledDiv>
+                    <StyledButton onClick={handleEdit}>Update Member</StyledButton>
+                </StyledDiv>
+                {editModal &&
+                    <MemberModal closeModal={closeModal}
```

Comment:
I think you could pass the handleEdit method to closeModal because it inverts the value of editModal.
Would make the closeModal method obsolete